cross-env development, replaced react-bootstrap for reactstrap

colindekker opened the pull request:
- made cross-env development ready in package.json (scripts, added rimraf),
- changed over react-bootstrap to reactstrap to support bootstrap 4.0.0-beta.2. 
- Removed imports for react-bootstrap (CHeckbox, Radio), migrated components to radio and checkbox inputs
- changed react import to 'import * as React'

ericgio commented:
Thanks for the PR @colindekker. There's a lot going on here; can you explain the purpose of these changes a little more please?

colindekker commented:
I've replaced react-bootstrap with reactstrap, a bootstrap 4 compatible wrapper so the component is futureproof. The Checkbox and Radio components do not exist in reactstrap so reverted those to basic checkboxes etc. (in the examples).
All React import are now 'import * as React', which seems to be the standard (at least as far as I can tell) in newer versions of React. components are the defined using React.Component. This is especially useful with Typescript.
Lastly, I changed the (buil) scripts to use cross env compatible commands so the project can be worked developed on Windows machines too.
I've added an NPM package based on this code if you want to check it out https://www.npmjs.com/package/reactstrap-typeahead

ericgio commented:
I'm updating the examples to support both BS3 and BS4. Better support for BS4 is being tracked in #131.
